lib/vmdb/settings.rb

```diff
@@ -6,9 +6,13 @@
 module Vmdb
   class Settings
     PASSWORD_FIELDS = %i(bind_pwd password amazon_secret).to_set.freeze
+    KNOCKOUT_STRING = "--!".freeze
```

Comment:
Yeah, wasn't sure what a "safe" string was for this.
I think the [deep merge default](https://github.com/danielsdeleo/deep_merge#selected-options-details) is just "--" which I figured wouldn't work for yml